x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -822,7 +822,11 @@ export const FIELD = (readOnly?: boolean): FieldMap => ({
           id="xpack.synthetics.monitorConfig.indexResponseHeaders.helpText"
           defaultMessage="Controls the indexing of the HTTP response headers to "
         />
-        <EuiCode>http.response.body.headers</EuiCode>
+        <EuiCode>
+          {i18n.translate('app_not_found_in_i18nrc.fIELD.http.response.body.headersCodeLabel', {
```

Comment:
weird translation title.

Reply:
I don't really think this needs to be translated. The ecs field doesn't change in different languages.